### DISCUSSION TITLE
Dart Sass Embedded binary during build

### DISCUSSION THREAD

**Discussion by brycewray** [vercel/community#420] [Idea]
For Sass/SCSS, the Hugo static site generator natively uses the [deprecated](https://sass-lang.com/blog/libsass-is-deprecated) LibSass implementation, but [Hugo 0.80+](https://github.com/gohugoio/hugo/releases/tag/v0.80.0) can support the recommended [Dart Sass](https://sass-lang.com/dart-sass) implementation **if** the [Dart Sass Embedded binary](https://github.com/sass/dart-sass-embedded) is in the path. This works fine in development, but obviously needs a host’s cooperation to work in a production build. Any chance Vercel will make this possible? The minimal request is for being able to specify the presence of the Dart Sass Embedded binary; the “nice-to-have” would be the capability to have that binary **and** specify its version. Presumably, both would involve some sort of environment variable; *e.g.*, either `DART_SASS_EMBEDDED` = `YES` or `DART_SASS_EMBEDDED` = `1.49.9` (that’s the current version at this writing).

Please refer also to this from the Hugo Discourse forum:
https://discourse.gohugo.io/t/using-dart-sass-hugo-and-netlify/37099/2

Thanks in advance for any consideration this request may receive.

cc: @bep

**Comment by bep** [comment]
Yes, this would be very much needed and appreciated by me and many.

**Reply by brycewray** [reply]
Any thoughts on this, @leerob ?
